### DISCUSSION TITLE
remoteEntry.js modules not to be cached

### DISCUSSION THREAD

**Discussion by aquarazorda** [module-federation/module-federation-examples#728] [Q&A]
Hello, I'm trying to load remoteEntry.js once and then use it to load same or different modules in different places, but the issue that I'm facing is that, modules get cached. So for example if I load the module second time, it has the state of first load and I can't avoid that.

With current loadRemoteModule() behavior, if I load same module with different data twice on the same page, I get two remoteEntry.js added to the DOM, so I overloaded it and if I already have remoteEntry.js present, I do not add it again. With this approach, I'm facing that module cache problem.

What I'm trying to achieve is, that I want to load remoteEntry.js once and use it to load modules in different places as many times as I want.

Thanks

**Comment by ScriptedAlchemy** [comment]
Inject the script with an ID and you can track if that script id exists, if so resolve to it else load the script tag

**Comment by ScriptedAlchemy** [comment]
The only way to prevent them from being cached would be to have the host require its own remote, so each invocation is an executed thunk
